substrate/frame/support/src/traits/tokens/asset_ops.rs

```diff
@@ -0,0 +1,238 @@
+//! Abstract asset operations traits.
```

Comment:
What do you think about using the word "intents" instead of "operations"?

Reply:
idk :)

However, there are a lot of places that need to be revisited to rename things, mostly in comments